Druid/DRBytecodeIRGenerator.class.st

```diff
@@ -232,7 +232,7 @@ DRBytecodeIRGenerator >> interpretInstructionPointerWith: aRBVariableNode [
 DRBytecodeIRGenerator >> interpretInterruptCheckWith: aRBMessageNode [
 
 	^ (self
-		   addInstructionWithNoResultFrom: aRBMessageNode
+		   addInstructionFrom: aRBMessageNode
```

Comment:
Here we are losing the "NoResult" is that ok?

Reply:
Yes, it has result